src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterKeyValuePair.cs

```diff
@@ -11,8 +11,11 @@ internal sealed class JsonKeyValuePairConverter<TKey, TValue> : JsonConverter<Ke
         private const string KeyName = "Key";
         private const string ValueName = "Value";
 
-        private static readonly JsonEncodedText _keyName = JsonEncodedText.Encode(KeyName);
-        private static readonly JsonEncodedText _valueName = JsonEncodedText.Encode(ValueName);
+        // "encoder: null" is used since the literal values of "Key" and "Value" should not normally be escaped
+        // unless a custom encoder is used that escapes these ASCII characters (rare).
+        // Also by not specifying an encoder allows the values to be cached statically here.
+        private static readonly JsonEncodedText _keyName = JsonEncodedText.Encode(KeyName, encoder: null);
```

Comment:
Out of scope of this PR, but we should pass in the user-defined encoder for this (and not null). Though unlikely, if a user creates an encoder that escapes everything, then the payload would be incorrect.